platform/o.n.swing.laf.flatlaf/src/org/netbeans/swing/laf/flatlaf/FlatLFCustoms.java

```diff
@@ -78,6 +79,9 @@ public Object[] createApplicationSpecificKeysAndValues() {
             // necessary for org.openide.explorer.propertysheet.PropertySheet and others
             CONTROLFONT, UIManager.getFont("Label.font"), // NOI18N
 
+            EXPLORER_FOLDER_ICON ,       ImageUtilities.loadImage("org/netbeans/swing/plaf/resources/hidpi-folder-closed.png"),
+            EXPLORER_FOLDER_OPENED_ICON, ImageUtilities.loadImage("org/netbeans/swing/plaf/resources/hidpi-folder-open.png"),
```

Comment:
I have a similar patch in my own NetBeans build, but in that case I used "Tree.closedIcon" and "Tree.openIcon" as the UIManager keys instead of "Nb.Explorer.Folder.icon" (EXPLORER_FOLDER_ICON) and "Nb.Explorer.Folder.openedIcon"(EXPLORER_FOLDER_OPENED_ICON). Perhaps the former is better, since it is probably applied in both the NetBeans explorer components as well as in the more general JTree component?

Reply:
Yeah, I checked this and the keys actually need to be "Tree.closedIcon" and "Tree.openIcon", otherwise in some cases you get a mix of folder icons like this:

<img width="314" alt="folder-karlspr" src="https://user-images.githubusercontent.com/886243/229313123-1b0f116f-a6c9-463c-8c0c-d74aebf0ae30.png">

(Probably because some parts of NetBeans explicitly looks for these icons in the JTree UIManager properties.)